packages/react-components/react-drawer/library/src/shared/drawerMotionUtils.test.tsx

```diff
@@ -4,7 +4,7 @@ import { DrawerMotionParams, InlineDrawerMotion } from './drawerMotions';
 import { mergePresenceSlots } from './drawerMotionUtils';
 
 const TestComponent = InlineDrawerMotion;
-const testProps: DrawerMotionParams = { position: 'start', size: 'medium' };
+const testProps: DrawerMotionParams = { position: 'start', size: 'medium', dir: 'ltr' };
```

Comment:
what is this adding to the test ? please add coverage that feature we implement works. ty

Reply:
There are two things here:
- This `dir: 'ltr'` is added here as the types were updated, so we need to mock the base motion params. This test only makes sure we are merging the props correctly.
- The overall fix cannot be tested correctly unfortunately. Since the screenshot tests cannot test animations and this is purely a CSS change.

I added a test for the specific function that generates the style.

Reply:
ty !